app/src/main/java/com/rob729/quiethours/Database/Profile.kt

```diff
@@ -14,6 +14,8 @@ data class Profile(
     var smin: Int,
     var ehr: Int,
     var emin: Int,
-    var d: String
+    var d: String,
+    // To store profile color in the database
+    var color_index: Int
```

Comment:
@coder2699 Just one small change.
Change the name format from the snake case to the camel case.
Rest looks good to me

Reply:
@Rob will the variable name "colorIndex" work?

Reply:
Yes

Reply:
@rob729 done 👍